Src/BootCamp.Chapter/CredentialsManager.cs

```diff
@@ -0,0 +1,130 @@
+        {
+            public Person(string name, string password)
+            {
+                Name = name;
+                Password = password;
+            }
+
+            public string Name { get; set; }
+            public string Password { get; set; }
+        }
+
+        internal static void Demo()
+        {
+            Console.WriteLine("Would you like to register or login");
+            Console.WriteLine("Choose R voor register");
+            Console.WriteLine("Choose L voor login");
+            var choice = Console.ReadLine();
+            switch (choice)
+            {
+                case "R":
+                    Register();
+                    break;
+
+                case "L":
+                    Login();
+                    break;
+
+                default:
+                    Console.WriteLine("This is not a right choice. Only R and L are valid");
+                    break;
+            }
+        }
+
+        private static void Login()
+        {
+            var isValidLogin = false;
+            var data = File.ReadAllText(@"Database/user.txt");
+            var fileData = data.Split($"{Environment.NewLine}");
+            var input = InputData();
+            bool isValidInput = TryParse(input, out Person personData);
+             if(!isValidInput)
+            {
+                Console.WriteLine("You entered invalid data");
+                return; 
+            }
+            for (int i = 0; i < 1; i++)
+            {
+                var person = fileData[i];
+                var splitted = person.Split(";");
+                if (splitted[1] == personData.Password & splitted[0] == personData.Name)
+                {
+                    isValidLogin = true;
+                }
+            }
+
+            Console.WriteLine(isValidLogin);
+        }
+
+        private static bool TryParse(string input, out Person person)
+        {
+            person = new Person(); 
+            if (String.IsNullOrEmpty(input))
+            {
+                return false; 
+            }
+
+            var parts = input.Split(" "); 
+
+            if (parts.Length != 2)
+            {
+                return false; 
+            }
+
+
+            var name = parts[0]; 
+            var password = Encoding.Unicode.GetBytes(parts[1]);
+            var encodedPassWord = Encoding.Unicode.GetString(password);
+
+            person = new Person(name, encodedPassWord);
+
+            return true;  
+        }
+
+        private static void Register()
+        {
+            var personData = InputData();
+            bool isValidInput = TryParse(personData, out Person person);
+            if(!isValidInput)
+            {
+                Console.WriteLine("You entered invalid data");
+                return; 
+            }
+            var output = $"{person.Name};{person.Password}{Environment.NewLine}";
+            File.AppendAllText(@"Database/user.txt", output);
+        }
+
+        private static String InputData()
```

Comment:
Having a second look, I get what you did there by calling this InputData. You ASK to input the data. This is ambigious, because I initially read this as Input of Data or data for input. In any case it's a vauge name.

PromptCredentials is what is actually happening here.

Reply:
renamed